test/cli/constant-fuzzy/constant-fuzzy.out

```diff
@@ -24,6 +44,18 @@ test/cli/constant-fuzzy/constant-fuzzy.rb:3: Unable to resolve constant `Intege`
 test/cli/constant-fuzzy/constant-fuzzy.rb:3: Unable to resolve constant `Int` https://srb.help/5002
      3 |Yieldr.foo(Intege, Int)
                            ^^^
+  Autocorrect: Use `-a` to autocorrect
+    test/cli/constant-fuzzy/constant-fuzzy.rb:3: Replace with `IO`
+     3 |Yieldr.foo(Intege, Int)
```

Comment:
The `Autocorrect` is duplicative of the `Did you mean`? one above. Should we remove the `Did you mean:`?